<launch>
  <arg name="Wakeup_Topic"/>
  <arg name="ASR_Topic"/>

  <node pkg="sphinx_wakeup" type="sphinx_wakeup_node" name="sphinx_wakeup_node" output="screen">
    <param name="sphinx_topic" value="$(arg Wakeup_Topic)" />
    <param name="asr_topic"    value="$(arg ASR_Topic)" />
    <param name="wakeup_word"  type="String" value="卡丁" />
  </node>
</launch>
